### PR TITLE
README.md: break up boot args into sections and sort them A-Z

### PR DESCRIPTION
**Pull request by perez987**
Minor changes to improve readability of the boot args table (in my opinion).

Anyway, thanks for your great work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,58 +45,67 @@ Read [FAQs](https://github.com/acidanthera/WhateverGreen/blob/master/Manual/) an
...
-| `-wegnoigpu` 	| `disable-gpu` property to IGPU 	| Disable internal GPU 	|
-| `-wegswitchgpu` 	| `switch-to-external-gpu` property to IGPU 	| Disable internal GPU when external GPU is installed 	|
-| `-radvesa` 	| N/A 	| Disable ATI/AMD video acceleration completely 	|
-| `-rad24` 	| N/A 	| Enforce 24-bit display mode 	|
-| `-raddvi` 	| N/A 	| Enable DVI transmitter correction (required for 290X, 370, etc.) 	|
-| `-radcodec` 	| N/A 	| Force the spoofed PID to be used in AMDRadeonVADriver 	|
-| `radpg=15` 	| N/A 	| Disable several power-gating modes (see [FAQ Radeon](https://github.com/dreamwhite/WhateverGreen/blob/master/Manual/FAQ.Radeon.en.md), required for Cape Verde GPUs) 	|
-| `agdpmod=vit9696` 	| `agdpmod` property to external GPU 	| Disable check for `board-id` 	|
-| `agdpmod=pikera` 	| `agdpmod` property to external GPU 	| Replaces `board-id` with `board-ix` 	|
+| Generic			    |
```
Comment (dreamwhite) [inline README.md:48]: I'd say to split the sections onto different tables using markdown `#`syntax

**Review by dreamwhite — CHANGES_REQUESTED** [review]

**Comment by dreamwhite** [comment]
@perez987 I proposed the change on perez987/WhateverGreen/pull/1 . Please give it a look

File: README.md (hunk 1 above)
Comment (perez987) [reply README.md]: @dreamwhite 
Done on my side, I have fixed board-id and 2nd stage boot headers.

**Comment by perez987** [comment]
@dreamwhite
Done on my side, I have fixed board-id and 2nd stage boot headers.

**Comment by Andrey1970AppleLife** [comment]
-cdfon for any gpu.

**Comment by perez987** [comment]
@Andrey1970AppleLife 
As far as I know, -cdfon has been related to iGPU in WhateverGreen readme and in Dortania GPU Buyers Guide but I’m not so experienced user to be able to agree or disagree to your comment. 
Can you explain a little more? Anyway it’s easy to move this boot arg to another section. 
Thanks.

**Comment by vit9696** [comment]
Yes, it has patches specific to different GPUs (e.g. NVIDIA), just it is much more common on IGPU. Should move to global section.

**Comment by perez987** [comment]
Changed Generic to Global.
Changed `-cdfon` from iGPU to Global.

**Comment by perez987** [comment]
Sorry for my lack of skills, I synced my repo with yours and PR closed automatically.
I reopened it.
Readme file has last been updated with last changes in [d93f49c](https://github.com/acidanthera/WhateverGreen/commit/d93f49c765cf27c1972bca4876c5094e71bbdd04).

**Pull request merged**